app/views/idv/shared/_verify.html.erb

```diff
@@ -0,0 +1,119 @@
+<%#
+locals:
+  pii - user's information
+  remote_identity_proofing - true if we're doing RIDP, false if we're doing IPP
+  step_url - generator for step URLs
+%>
+<% title t('titles.doc_auth.verify_info') %>
+
+<%= render PageHeadingComponent.new.with_content(t('doc_auth.headings.verify')) %>
+<div class='margin-top-4 margin-bottom-2'>
+  <% if remote_identity_proofing %>
+    <div>
+      <%= "#{t('doc_auth.forms.first_name')}: #{pii[:first_name]}" %>
```

Comment:
So I know it's copy-pasted from the other page, but IMO this kind of interpolation is very silly `#{}` for something trivial inside of `<%= `, if it's not too much effort, I'd consider breaking these out:

```suggestion
      <%= t('doc_auth.forms.first_name')} %>: <%= pii[:first_name] %>
```

Reply:
I've also been eyeing these for some time wondering if there's some more semantic HTML we could be using for this `field: value` association. With the barcode attention page in #6483, I went with a [`dl`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) for a similar listing ([screenshot](https://user-images.githubusercontent.com/1779930/174330301-c05c3540-88c0-407a-a7b5-3618f492b7a5.png)):

https://github.com/18F/identity-idp/blob/2979497b0f60f0acaa042bbf6655742fd9cc5b5c/app/javascript/packages/document-capture/components/barcode-attention-warning.tsx#L58-L71

Reply:
I'm up to consider these and use one or the other of these suggestions but I'd prefer to do it in a separate PR. I'm trying to keep this PR's changes to existing prod behavior minimal and then follow-up with a couple PRs that make more specific changes to existing prod behavior